src/platform_impl/web/window.rs

```diff
@@ -195,10 +208,26 @@ impl Inner {
 
     #[inline]
     pub fn set_cursor_icon(&self, cursor: CursorIcon) {
-        *self.previous_pointer.borrow_mut() = cursor.name();
+        *self.selected_cursor.borrow_mut() = SelectedCursor::BuiltIn(cursor);
+        if let Some(s) = backend::get_canvas_style_property(self.canvas.borrow().raw(), "cursor") {
+            if s == "none" {
+                return;
+            }
```

Comment:
This is new behavior, which should probably be documented.
Could you explain the thought behind it though? It seems fine to me to overwrite any set cursors if the user wants to. It's explicit after all.

Reply:
I think the issue was that setting a new cursor was overriding the hidden status, which is different from other platforms. It could be another issue though because it was broken before this pr.

Reply:
Ah, if that's the case we should probably use internal information to determine that instead of reading the CSS style. But preferably just leave it out of this PR and open a separate issue.